### ISSUE TITLE
aria-required-children broken for Date picker component

### ISSUE THREAD

**Issue by AkshanshSrivastava03** [OfficeDev/office-ui-fabric-react#3980]
### Bug Report

- __Package version(s)__: 5.46.1002
- __Browser and OS versions__: Chrome

#### Describe the issue:

aria-required-children is not correctly instantiated for Date picker component.

#### Actual behavior:

![image](https://user-images.githubusercontent.com/24951429/36252174-b59053ec-1269-11e8-9e0b-7aed8a40e705.png)

#### Expected behavior:

This should be correctly used according to aria-design patterns for Menu

**Comment by Jahnp** [comment]
@johannao76, could you or someone from your team take a look at this when you have a chance?

**Comment by johannao76** [comment]
Adding @John Lorenz<mailto:jolore@microsoft.com>
________________________________
